test/e2e/util.go

```diff
@@ -161,6 +161,10 @@ type TestContextType struct {
 	GatherMetricsAfterTest            bool
 	// Currently supported values are 'hr' for human-readable and 'json'. It's a comma separated list.
 	OutputPrintType string
+	// CreateTestingNS is responsible for creating namespace used for executing e2e tests.
+	// It accepts namespace base name, which will be prepended with e2e prefix, kube client
+	// and labels to be applied to a namespace.
+	CreateTestingNS func(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error)
```

Comment:
So how are you expecting the custom namespace function to be used? Is it going to do something and then call the `test/e2e/util.go` version? Or will it fully supersede it?